.env

```diff
@@ -22,6 +22,8 @@ HASH_KRATOS_PG_PASSWORD=kratos
 HASH_KRATOS_PG_DEV_DATABASE=dev_kratos
 HASH_KRATOS_PG_TEST_DATABASE=test_kratos
 
+HASH_GRAPH_LOG_LEVEL=info
```

Comment:
I know we discussed this yesterday, but does this end up used for the development docker compose too? Perhaps we just want to change the defaults in the prod docker compose file, and leave this undefined here

Reply:
It would take precedence over the dockerfile defaults, yes

Reply:
I think we should probably still have more in-depth logs in trace mode, just not in the prod docker compose

Reply:
In that case I'll revert the commit adding this .env value

Reply:
Can you also update 
```rml
      RUST_LOG: "${HASH_GRAPH_LOG_LEVEL:-graph=trace,hash_graph=trace,tokio_postgres=debug}"
```
in `docker-compose.prod.yml`

Reply:
Maybe to 
```yml
      RUST_LOG: "${HASH_GRAPH_LOG_LEVEL:-info}"
```

Reply:
If that's the right syntax 😅  (not sure)

Reply:
169d82b97c245a57a4cd663902b8ff2e0ccdd37e and 4d014730279915ac1601e7aa51293320330c0ef9